### PR TITLE
Only require one shipit for work in linkerd/admin

### PR DESCRIPTION
**Pull request by rmars**
Pullapprove documentation here:
http://docs.pullapprove.com/yaml/reviewers/

I /think/ this is what we want (?)

### CODE REVIEW

**Comment by adleong** [comment]
:star: 🎣

File: .pullapprove.yml [hunk 1]
```diff
@@ -3,14 +3,30 @@ approve_regex: '^(Approved|lgtm|LGTM|:shipit:|:star:|:\+1:|:ship:)'
...
+    - adleong
+    - esbie
+    - gtcampbell
+    - klingerf
+    - olix0r
+    - rmars
+    - siggy
+    - stevej
+    name: linkerd-admin
+    required: 1
```
Comment (adleong) [inline .pullapprove.yml]: this is crazy, but what happens if you put `-1` here?

Comment (rmars) [inline .pullapprove.yml]: Then pullapprove requires approval from /all/ members :(

`A value of -1 means that approval is required from all members.`

**Comment by rmars** [comment]
There's no way to apply exclusions currently.  There is an open issue in pullapprove:
https://github.com/pullapprove/support/issues/52

**Comment by rmars** [comment]
Won't fix for now...

**Pull request closed without merging**